src/components/Components/GroupItem.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+
+const GroupItem = ({ children, key }) => (
+  <li
+    key={key}
```

Comment:
Nit:  I might be mistaken, but I don't think you need the `key` prop here. It's only important to have it on the component that is created within a loop, not so much in the render itself.

(for more infos: https://overreacted.io/react-as-a-ui-runtime/#lists)